SR2025/2024-09-18-secondary-contact-info.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Student Robotics 2025 secondary contacts, CC primary contacts
+subject: "Student Robotics 2025: you are a secondary contact"
+---
+
+Hi,
+
+As you're hopefully aware from $TODO_PRIMARY_CONTACT_FULL_NAME,
+[Student Robotics][student-robotics] is an exciting competition for
+16-19 year-olds which a team from $TODO_ORGANISATION is taking part in.
+
+$TODO_PRIMARY_CONTACT_GIVEN_NAME gave us your details as a secondary contact at
+$TODO_ORGANISATION in the unlikely case that we are unable to contact them.
+Please speak to them in the first instance if you have any questions.
+
+As [team supervisor][team-supervisor], $TODO_PRIMARY_CONTACT_GIVEN_NAME is
+responsible for your competitors as well as the [kit][kit] which we loan to each
+team. Our communication with your team will also all be sent to them.
```

Comment:
There's two things I'm trying to convey here:
- awareness of the kit loan
- awareness that if the primary contact disappears that we'll chase the secondary for the kit and for looking after the competitors

I almost want to make it clear that the secondary is thus also responsible for the kit, but I'm not sure if there's a way to do that without it getting really complicated.